update-testsuite.sh

```diff
@@ -128,8 +128,16 @@ for repo in ${repos}; do
         popdir
     fi
 
+    # Check whether any files were removed.
+    for old in $(find ${wast_dir} -maxdepth 1 -name \*.wast); do
+      new=$(find repos/${repo}/test/core -name ${old##*/})
+      if [[ ! -f ${new} ]]; then
+          log_and_run git rm ${old}
+      fi
+    done
+
     # Check whether any files were updated.
-    if [ $(git status -s ${wast_dir}/*.wast | wc -l) -ne 0 ]; then
+    if [ $(git status -s ${wast_dir} | wc -l) -ne 0 ]; then
         log_and_run git add ${wast_dir}/*.wast
```

Comment:
if you are going to change the status line above dont you also want to change this line?

Reply:
I thought so, but it seems to work OK... I'm also a little worried about adding random loose files from the directory. Is there a better way to do this?

Reply:
But I guess my question is why is this change needed?

Reply:
Oh, because I wanted to include the changed repo, even if all that happens is removal of `.wast` files. Without this, it will remove the files but it won't be in the change description.

Reply:
Oh interesting.     I guess you could have also put "${wast_dir}/*.wast" in quotes to make git (and not bash) do the expansion and include files that don't exist on disk anymore.